src/main/scala/ing/wbaa/druid/definitions/QueryContext.scala

```diff
@@ -0,0 +1,69 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package ing.wbaa.druid.definitions
+
+object QueryContext {
```

Comment:
We should update this one for each version of Druid. For example, with the new vectorized queries, the `vectorSize` is introduced: https://github.com/apache/incubator-druid/blob/master/processing/src/main/java/org/apache/druid/query/QueryContexts.java#L159

Would be great to fetch the settings from the Druid codebase, but I don't know how to do that easily.

Reply:
I've added `vectorize` and `vectorSize` parameters. 

You are right, it would be useful to have some (semi-)automated way to export all these parameters. On the other hand, context in Scruid is represented by a generic Map[String, String] and can still be used in cases that we do not have the context parameter key in the `QueryContext` object.

Reply:
Perfect, thanks!